NEWS.md

```diff
@@ -3,9 +3,9 @@
 ## Bootstrap 5
 
 * pkgdown can style your site with Bootstrap 5 (with help from @jayhesselberth,
-  @apreshill, @cpsievert). Opt-in by setting `boostrap` version in your 
+  @apreshill, @cpsievert). Opt-in by setting `boostrap` version in your
```

Comment:
Can you please eliminate the spurious diffs?

Reply:
Should be fixed now